tox.ini

```diff
@@ -0,0 +1,49 @@
+[tox]
+envlist = py-unittest
+indexserver =
+    preinstall = https://pypi.python.org/simple
+
+[base]
+deps =
+    :preinstall: -rrequirements.txt
```

Comment:
Is it right that there's no space between -r and requirements?

Reply:
I just followed their convention from examples. No space is needed apparently.